### PR TITLE
Redesign the curriculum page

### PR DESCRIPTION
**Pull request by arun1595**
## Screenshots

Course card:
- **Incomplete state**

image

- **Completed state**

image

### CODE REVIEW

File: app/views/courses/_course_description.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <%= image_tag course.badge, alt: "#{course.title} badge", class: 'course-badge' %>
+  </div>
+
+  <div class="col-sm-9 course-title-wrapper">
+    <div class="course-title">
+      <%= link_to course do %>
+        <h2 class="text-uppercase">
+          <%= course.title %>
+          <% unless course.is_active %>
+            <span class="coming-soon"> ...Coming Soon!</span>
```
Comment (arun1595) [inline app/views/courses/_course_description.html.erb]: @KevinMulhern Do we need this?

File: app/views/courses/_course_overview.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<div class="projects">
+  <h4>Projects</h4>
+
+  <ul class="projects-list">
+    <% course.you_build.each do |project| %>
+      <li><%= project %></li>
+    <% end %>
+  </ul>
+
+  <%= render 'see_more_link', course: course %>
```
Comment (arun1595) [inline app/views/courses/_course_overview.html.erb]: @KevinMulhern In the current design(production version), we display the 'See More' links twice. Do we _really_need it to be displayed twice?

If not, I will move the partial `_see_more_link` to `_course_overview` partial.

File: app/views/courses/index.html.erb [hunk 3]
```diff
@@ -1,100 +1,26 @@
...
     <div class="heading-box">
       <div class="heading">
-        <h1>This is Your Path to Learning Web Development</h1>
-      </div>
-      <div class="subheading">
+        <h1 class="text-center camel light curriculum-title">Curriculum</h1>
+      </div> <!-- /.heading -->
+
+      <div>
         <p>
```
Comment (arun1595) [inline app/views/courses/index.html.erb]: @KevinMulhern Do we need to include this in the curriculum page?

File: app/views/courses/_course_description.html.erb (hunk 1 above)
Comment (KevinMulhern) [inline app/views/courses/_course_description.html.erb]: I don't think so, delete it 😄

File: app/views/courses/_course_overview.html.erb (hunk 2 above)
Comment (KevinMulhern) [inline app/views/courses/_course_overview.html.erb]: If its only displayed once in the new version, theres no need to display it twice.

File: app/views/courses/_course_description.html.erb (hunk 1 above)
Comment (arun1595) [inline app/views/courses/_course_description.html.erb]: :+1:

File: app/views/courses/index.html.erb (hunk 3 above)
Comment (KevinMulhern) [inline app/views/courses/index.html.erb]: the <p>?

Comment (arun1595) [inline app/views/courses/index.html.erb]: Lol. My bad. I meant to ask if we need the couple of paragraphs

Comment (KevinMulhern) [inline app/views/courses/index.html.erb]: ah I see 😄 

I would make it look as close to the design as possible, We will probably be updating the copy on this page as well. 

It looks there are no "see more" links in the redesign mock up so I would take them out completley.

Comment (arun1595) [inline app/views/courses/index.html.erb]: Okay. Let me remove them.

**Comment by KevinMulhern** [comment]
👍 LGTM

**Pull request merged**